### DISCUSSION TITLE
How do you hide the rich text field toolbar

### DISCUSSION THREAD

**Discussion by Deanobeano2** [cortexrd/Knack-Toolkit-Library#332] [Q&A]
Hi

I've seen this question in the Knack forum but without an answer supplied and wondered if this can be achieved using the ktlVisibilityHidden keyword. Tinkering around and changing the redeactor-toolbar element.style visibility from visible to hidden appears to achieve the desired result.

How would you achieve this using the ktlVisibilityHidden keyword?

Thanking you.
Dean

**Comment by cortexrd** [comment]
If you use the ktlVisibilityHidden class, it will works, but leave an empty white space.

To get a clean layout, this is the right way to do it in the console:
`$('#view_567 #kn-input-field_227 .redactor-toolbar').addClass('ktlDisplayNone')`

With a KTL keyword, it would look like this:
`_cls=[ktlDisplayNone], [ktlTarget, $('#view_567 #kn-input-field_227 .redactor-toolbar')]`

BTW... can you give us that link to the forum question?

Norm

**Reply by Deanobeano2** [reply]
Norm

Thank you very much. That generated a better/cleaner looking result than I was expecting from my tinkering.

Unfortunately, I can't find the question posed on the forum but from memory it was dated in 2022, I believe, and received no responses. I was thinking of posting in the Knack forum about a web page I discovered which resolved an issue, for a small image/PDF, I encountered when following Carl's method for generating the embedding code and would be happy to mention your quick and easy solution to make the field appear just as a picture.

To complete the appearance of the field as a picture can the keyword also be used to prevent the inner light grey border from appearing around the image on hover?
![Rich Text field - border appears on hover](https://github.com/user-attachments/assets/947b584b-9633-4064-a902-cedadef62929)

Dean
